pkg/format/format.go

```diff
@@ -23,18 +23,40 @@ const (
 var (
 	labelVar  string
 	fsTypeVar string
+	deviceVar string
 	drives    map[string]bool
 	driveKeys []string
 )
 
 func autoformat(label, fsType string) error {
 	var first string
 	for _, d := range driveKeys {
-		err := exec.Command("sfdisk", "-d", d).Run()
+		log.Printf("Considering auto format for device %s", d)
+
+		out, err := exec.Command("blkid", d).Output()
+		if err == nil {
+			log.Printf("Skipping autoformat of %s. blkid returned: %s", d, out)
+			continue
```

Comment:
if `blkid` returns with exitcode 0, then there was content detected on the device; skip the autoformat, and also include the uuid / label / type which was found, as applicable